Source/ConversationV1/Conversation.swift

```diff
@@ -1503,20 +1501,20 @@ public class Conversation {
      - parameter workspaceID: The workspace ID.
      - parameter entity: The name of the entity.
      - parameter value: The text of the entity value.
-     - parameter properties: An UpdateValue object defining the new content for value for the entity.
+     - parameter body: An UpdateValue object defining the new content for value for the entity.
```

Comment:
Same with this `UpdateValue` model, which has an `x-alternate-name` of `properties`.